queries/useQueryPools.ts

```diff
@@ -54,6 +54,7 @@ export type PoolLiquidityState = {
 
 export type PoolEntityTypeWithLiquidity = PoolEntityType & {
   liquidity: PoolLiquidityState
+  swap_info: Awaited<ReturnType<typeof querySwapInfo>>
```

Comment:
hmm... `typeof querySwapInfo` returns a string, let's double check if we want to add some typing to querySwapInfo to improve the type here